fastlane_bot/events/managers/base.py

```diff
@@ -321,6 +321,7 @@ def get_rows_to_update(self, update_from_contract_block: int) -> List[int]:
             for i, pool_info in enumerate(self.pool_data)
             if pool_info["last_updated_block"]
             < update_from_contract_block - self.alchemy_max_block_fetch
+            or str(pool_info["last_updated_block"]) == "nan"
```

Comment:
This cast is dangerous.

Consider one of the following:
1. Converting that string to lowercase
2. Using `math.isnan` instead
3. Using `np.isnan()` instead
4. Using `pool_info["last_updated_block"] != pool_info["last_updated_block"]` instead